### ISSUE TITLE
Resource viewer loads forever for sidecar containers

### ISSUE THREAD

**Issue by GuessWhoSamFoo** [vmware/octant#83]
Example config:

```
apiVersion: apps/v1
kind: Deployment 
metadata:
  name: sidecar
spec:
  selector:
    matchLabels:
      app: sidecar
  replicas: 2
  template:
    metadata:
      labels:
        app: sidecar
    spec:
      volumes:
      - name: shared-logs 
        emptyDir: {}
    
      containers:
      - name: app-container
        image: alpine
        command: ["/bin/sh"]
        args: ["-c", "while true; do date >> /var/log/app.txt; sleep 5;done"]
    
        volumeMounts:
        - name: shared-logs
          mountPath: /var/log
    
      - name: sidecar-container
        image: nginx:1.16.0
        ports:
          - containerPort: 80
    
        volumeMounts:
        - name: shared-logs
          mountPath: /usr/share/nginx/html
```

**Comment by bryanl** [comment]
I'm looking at this in our 1.15 / resource viewer branch, and I'm not seeing it as broken:

![Screenshot from 2019-07-27 18-04-45](https://user-images.githubusercontent.com/240/62000942-19bfdd00-b099-11e9-823d-c0ab8b823c38.png)

Could you try this with the code in #85? I did see another issue:

![Screenshot from 2019-07-27 18-06-23](https://user-images.githubusercontent.com/240/62000946-4ecc2f80-b099-11e9-8860-3791e71464c5.png)

There are two containers, but the pod is only showing one. The pod template in the deployment looks correct.

![Screenshot from 2019-07-27 18-07-33](https://user-images.githubusercontent.com/240/62000952-82a75500-b099-11e9-9162-2ffe5d242bba.png)

**Comment by GuessWhoSamFoo** [comment]
This will be fixed via #85. I'll take a look to see if fixing containers can be squeezed in before 0.4.1
